sdk-ble/build.gradle

```diff
@@ -15,4 +15,6 @@ compileKotlin {
 
 dependencies {
     api project(":sdk")
+    api project(":sdk-ftms")
```

Comment:
The `sdk-ble` module is meant as a module that is solely used for the original Bike BLE implementation, built on top of `sdk`.
When adding support for newer devices (like now with FTMS), it's better to keep them completely separated, and form a dependency tree like so:

```
+----------+
| sdk-ble  |
+----------+
  |
  |
  v
+----------+
|   sdk    |
+----------+
  ^
  |
  |
+----------+
| sdk-ftms |
+----------+
```

Reply:
On second thought, since the FTMS protocol calculates everything by itself, it shouldn't even have a dependency on `sdk`? 
If you want to reuse domain modules, extract them out of `sdk` into a separate shared module.

Reply:
Thats a good idea